app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/WindowWidget.java

```diff
@@ -605,7 +607,12 @@ public void onTextPrompt(GeckoSession session, String title, String msg, String
 
     @Override
     public void onAuthPrompt(GeckoSession session, String title, String msg, AuthOptions options, AuthCallback callback) {
```

Comment:
re: the issue with the truncated message ([Gecko platform bug 1533468](https://bugzilla.mozilla.org/show_bug.cgi?id=1533468)): I tried to investigate, but no debug-log messages appeared when I called `adb logcat | grep -i onAuthPrompt` when going through a [Basic-Auth flow](https://webxr.sh/basic-auth#link):

```diff
diff --git a/app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/WindowWidget.java b/app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/WindowWidget.java
index 8fb90ee..b494fb4 100644
--- a/app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/WindowWidget.java
+++ b/app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/WindowWidget.java
@@ -548,7 +548,7 @@ public class WindowWidget extends UIWidget implements SessionStore.SessionChange
         GeckoSession session = SessionStore.get().getSession(mSessionId);
         return (session != null) && session.getTextInput().onKeyMultiple(aKeyCode, repeatCount, aEvent);
     }
-    
+
     @Override
     protected void onFocusChanged(boolean aGainFocus, int aDirection, Rect aPreviouslyFocusedRect) {
         super.onFocusChanged(aGainFocus, aDirection, aPreviouslyFocusedRect);
@@ -607,6 +607,7 @@ public class WindowWidget extends UIWidget implements SessionStore.SessionChange
 
     @Override
     public void onAuthPrompt(GeckoSession session, String title, String msg, AuthOptions options, AuthCallback callback) {
+        Log.d(LOGTAG, "onAuthPrompt: title:[" + title.toString() + "] msg:[" + msg.toString() + "]");
         mAuthPrompt = new AuthPromptWidget(getContext());
         mAuthPrompt.mWidgetPlacement.parentHandle = getHandle();
         mAuthPrompt.setTitle(title);
```

```diff
diff --git a/app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/WindowWidget.java b/app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/WindowWidget.java
index 8fb90ee..efeaf3e 100644
--- a/app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/WindowWidget.java
+++ b/app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/WindowWidget.java
@@ -548,7 +548,7 @@ public class WindowWidget extends UIWidget implements SessionStore.SessionChange
         GeckoSession session = SessionStore.get().getSession(mSessionId);
         return (session != null) && session.getTextInput().onKeyMultiple(aKeyCode, repeatCount, aEvent);
     }
-    
+
     @Override
     protected void onFocusChanged(boolean aGainFocus, int aDirection, Rect aPreviouslyFocusedRect) {
         super.onFocusChanged(aGainFocus, aDirection, aPreviouslyFocusedRect);
@@ -607,6 +607,7 @@ public class WindowWidget extends UIWidget implements SessionStore.SessionChange
 
     @Override
     public void onAuthPrompt(GeckoSession session, String title, String msg, AuthOptions options, AuthCallback callback) {
+        Log.d(LOGTAG, "called onAuthPrompt method");
         mAuthPrompt = new AuthPromptWidget(getContext());
         mAuthPrompt.mWidgetPlacement.parentHandle = getHandle();
         mAuthPrompt.setTitle(title);
```

any ideas why the debug-log message is not appearing when I call `adb logcat | grep -i onAuthPrompt`?